clinica/iotools/utils/data_handling.py

```diff
@@ -696,7 +720,7 @@ def create_subs_sess_list(
             session_df = pd.read_csv(
                 path.join(sub_path, subj_id + "_sessions.tsv"), sep="\t"
             )
-            session_list = list(session_df["session_id"].to_numpy())
+            session_list = list(session_df["session_id"].dropna().to_numpy())
```

Comment:
How come that we can get n/a values for the `session_id` here?

Reply:
It should not be the case Indeed.